Cargo.lock

```diff
@@ -2178,6 +2179,12 @@ version = "0.1.9"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a3e378b66a060d48947b590737b30a1be76706c8dd7b8ba0f2fe3989c68a853f"
 
+[[package]]
+name = "md5"
```

Comment:
Do we not already have md5 somewhere in our binary?

Reply:
Surprisingly, no.